app/assets/javascripts/admin/routes/admin-users-list-show.js.es6

```diff
@@ -1,17 +1,26 @@
-import AdminUser from "admin/models/admin-user";
-
 export default Discourse.Route.extend({
-  model: function(params) {
-    this.userFilter = params.filter;
-    return AdminUser.findAll(params.filter);
+  queryParams: {
+    order: { refreshModel: true },
+    ascending: { refreshModel: true }
   },
 
-  setupController: function(controller, model) {
-    controller.setProperties({
-      model: model,
-      query: this.userFilter,
-      showEmails: false,
-      refreshing: false
-    });
+  beforeModel(transition) {
```

Comment:
This is questionably not the best solution but changing this would I think  require an heavy refactoring of how admin-users-list-show is working. I think this is a good tradeoff.

Reply:
It's not ideal, but I am cool with it as a compromise. I would suggest that you add a comment in the code explaining why it's done this way though for future developers.